packages/api/src/utils/logParser.ts

```diff
@@ -233,13 +251,13 @@ abstract class ParsingInterface<T> {
   abstract _parse(
     log: T,
     ...args: any[]
-  ): LogStreamModel | MetricModel | RrwebEventModel;
+  ): Promise<LogStreamModel | MetricModel | RrwebEventModel>;
 
-  parse(logs: T[], ...args: any[]) {
+  async parse(logs: T[], ...args: any[]) {
     const parsedLogs: any[] = [];
```

Comment:
Can we type this `parsedLogs` here ? I think that's the reason why ts didn't pick it up. Also its better to double check all any types in this file